src/com/base/game/combat/SpecialAttack.java

```diff
@@ -997,6 +998,61 @@ public boolean isConditionsMet(GameCharacter owner) {
 		}
 	},
 
+	COW_GORE(50, "Gore", "HornIcon", Colour.DAMAGE_TYPE_PHYSICAL, DamageType.PHYSICAL, DamageLevel.HIGH, DamageVariance.LOW, SpecialAttackSpellCosts.MEDIUM, Util.newHashMapOfValues(new Value<StatusEffect, Integer>(StatusEffect.CRIPPLE, 4))) {
+		@Override
+		public String applyEffect(GameCharacter caster, GameCharacter target, boolean isHit, boolean isCritical) {
+
+			float damage = calculateDamage(caster, target, isCritical), cost = calculateCost(caster);
+
+			descriptionSB = new StringBuilder();
+			
+			if (caster == Main.game.getPlayer()) {
+				descriptionSB.append(UtilText.genderParsing(target,
+						"<p>" + "With a burst of energy, you leap forwards, trying to ram your horns at " + target.getName("the") + "."
+								+ (isHit ? " Your cow-like horns slam into " + target.getName("the") + "'s body,"
+										+ " and you manage to cause some serious damage with your sharp horns before <she> manages to throw you off of <herPro>."
+										: target.getName("The") + " manages to jump to one side, and there's an audible woosh as your horns move through the thin air.")
+								+ "</p>")
+						+ getDamageAndCostDescription(caster, target, cost, damage, isHit, isCritical));
+			} else {
+				descriptionSB.append(UtilText.genderParsing(caster,
+						"<p>" + "With a sudden burst of energy, " + caster.getName("the") + " leaps forwards as <she> tries to gore you."
+								+ (isHit ? " <Her> cow-like muzzle slams down on your body,"
+										+ " and <she> shakes <her> head from side-to-side, managing to cause some serious damage with <her> sharp horns before you manage to throw <herPro> off of you."
+										: "You jump to one side as you see the attack coming, and there's an audible woosh as <her> horns hit nothing but thin air.")
+								+ "</p>")
+						+ getDamageAndCostDescription(caster, target, cost, damage, isHit, isCritical));
+			}
+			
+			// If attack hits, apply damage and effects:
+			if (isHit) {
+				descriptionSB.append(target.incrementHealth(-damage));
+				for (Entry<StatusEffect, Integer> se : getStatusEffects().entrySet())
+					target.addStatusEffect(se.getKey(), se.getValue());
+			}
+
+			caster.incrementStamina(-cost);
+			
+			return descriptionSB.toString();
+		}
+
+		@Override
+		public String getDescription(GameCharacter owner) {
+			if (owner.isPlayer())
+				return "Your anthropomorphic cow-like horns can be used to deliver a powerful attack.";
+			else
+				return UtilText.genderParsing(owner, owner.getName("The") + "'s anthropomorphic cow-like horns can be used to deliver a powerful attack.");
+		}
+
+		@Override
+		public boolean isConditionsMet(GameCharacter owner) {
+			if (owner.getHornType() == HornType.BOVINE_MALE)
```

Comment:
replace with "return owner.getHornType() == HornType.BOVINE_MALE;"

Reply:
Not completely sure about this one, as it matches the style of the rest